assets/chat/js/user.js

```diff
@@ -53,12 +53,12 @@ class ChatUser {
       this.displayName = user;
       this.username = this.displayName.toLowerCase();
     } else {
-      this.id = user.id || null;
-      this.displayName = user.nick || '';
+      this.id = user?.id || null;
+      this.displayName = user?.nick || '';
       this.username = this.displayName.toLowerCase();
-      this.createdDate = user.createdDate || '';
-      this.features = user.features || [];
-      this.watching = user.watching || null;
+      this.createdDate = user?.createdDate || '';
+      this.features = user?.features || [];
+      this.watching = user?.watching || null;
```

Comment:
Do we still need to be nullsafe here considering "userless" broadcasts are now sent by the system user instead?

Reply:
Yeah, not necessary - didn't catch it because I had the old user-less messages in history. TY!